packages/react-slider/src/components/Slider/useSliderStyles.ts

```diff
@@ -188,13 +192,75 @@ const useTrackStyles = makeStyles({
   }),
 });
 
+/**
+ * Styles for the mark slot
+ */
+const useMarksWrapperStyles = makeStyles({
+  marksWrapper: theme => ({
+    position: 'relative',
+    display: 'grid',
+    outline: 'none',
+    zIndex: '1',
+    whiteSpace: 'nowrap',
+    '& .ms-Slider-mark': {
```

Comment:
I'm not seeing `ms-Slider-mark` applied anywhere? Also it seems like in converged it's more common to have a separate style `mark` and apply that where needed rather than hardcoding class names.

Reply:
@czearing would you mind maybe moving that to a const in the other file and importing it here? Or maybe putting the const in here and importing it in `useSliderState` might be the better thing to do? Either way, you'd have to individually specify the exports from these files in `index.ts` to ensure you're not exporting something you don't want to.